frontend/src/components/Carte.jsx

```diff
@@ -1,31 +1,25 @@
-import React, { useState, useEffect } from "react";
-
-function Carte() {
-  const [cardList, setCardList] = useState([]);
-  /* This is fetch API */
-  useEffect(() => {
-    fetch("https://api.punkapi.com/v2/beers?brewed_before=11-2012&abv_gt=6")
-      .then((response) => response.json())
-      .then((data) => setCardList(data.slice(0, 5)));
-  }, []);
+import PropTypes from "prop-types";
 
+function Carte({ handcard }) {
```

Comment:
if that's the one to be kept, rename it in english :)

handcard is a strange name (and is not in camelCase): cardData ? data ? card ?